tests/expected_results/end-to-end/reg_log

```diff
@@ -0,0 +1,11 @@
+INFO:2020-01-30 14:36:42,958:AtacWorks-calculate-metrics] Loading intervals
```

Comment:
same, rename the file to be more descriptive of the content

Reply:
this isn't needed anymore. removed it.